### PR TITLE
Updated DG, main sequence diagrams, fixed various diagrams, and inserted my PP

### PR DESCRIPTION
**Pull request by SimJunYou**
Made some fixes for the stuff Prof pointed out in issue #147 and ticked the relevant checkboxes there.

Updated, neatened, and clarified other diagrams by adding notes and stuff.

Added my PPP draft to the folder.

Also made Ravi a static class because why did we even make it non-static in the first place???

### CODE REVIEW

File: docs/team/f0fz.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+#### Contributions to documentation
+
+* DG:
+    * Wrote the following sections: Setting Up, Value Proposition, Implementation.
+    * Added class diagrams for the Main Class, and the Command/Data/Parser/TimeTable Family Classes.
+    * Added sequence diagram for the main loop.
+
+* UG:
+    *
```
Comment (tobiasceg) [inline docs/team/f0fz.md:52]: i think u missed out this part

**Review by tobiasceg — APPROVED** [review]
PPP still a draft but looks good

File: docs/team/f0fz.md (hunk 1 above)
Comment (SimJunYou) [reply docs/team/f0fz.md:52]: Because I haven't actually done anything for UG :x

Comment (tobiasceg) [reply docs/team/f0fz.md:52]: i guess u can say coz u arent a user and u dont need the guide 💯 😆

**Pull request merged**